modules/boards/spec/features/action_boards/version_board_spec.rb

```diff
@@ -156,14 +156,15 @@ def create_new_version_board
       subjects = WorkPackage.where(id: second.ordered_work_packages.pluck(:work_package_id)).pluck(:subject, :version_id)
       expect(subjects).to match_array [['Task 1', other_version.id]]
 
-      # Expect that version is not available for global filter selection
-      filters.expect_available_filter 'Version', present: false
```

Comment:
The selector for `expect_available_filter` was wrong, causing it to never find an element, and thus this check always returned true.